### PR TITLE
Sprint 2/issues 4#home

### LINKED ISSUE #6: sprint-2/issues-4#Home
## REQUIREMENTS / DESCRIPTION

> Create the Home screen, which will show jobs as well as companies in your city along with a search bar along with a navigation and also a small welcome message.
## DESIGN

image

### PR DESCRIPTION
**Pull request by Jherrera11**
# Description

What did you implemented/Why did you implemented this:

-Add Home screen
-Add the Interface folder
-Add the data folder 
-Add Card Company Component
-Add Card Work Component
-Add Imagenes 

Examples:

null

## Testing

null
 Example:

null

## Screenshots

image

### CODE REVIEW

**Review by VISOSO2403 — CHANGES_REQUESTED** [review]
pass the images through Squoosh to reduce their size and the svg files pass them through SVGOMG to reduce their size and make the application more fluid.

**Review by FelixAva — CHANGES_REQUESTED** [review]
You have double blank spaces, delete them.
You must put the correct indentation.
Use the close label if you really need it

**Review by FelixAva — APPROVED** [review]
LGTM

File: src/assets/user-male-avatar.jpg
Comment (VISOSO2403) [inline src/assets/user-male-avatar.jpg:1]: This image is jpg, change to webp

File: src/assets/wallpainter.jpg
Comment (VISOSO2403) [inline src/assets/wallpainter.jpg:1]: This image is jpg, change to webp

**Review by VISOSO2403 — CHANGES_REQUESTED** [review]
only changes those images, everything else looks fine.

**Review by VISOSO2403 — APPROVED** [review]
LGTM

**Pull request merged**